src/osemgrep/cli_scan/Rule_filtering.ml

```diff
@@ -15,12 +15,30 @@
 (*****************************************************************************)
 (* Types *)
 (*****************************************************************************)
-type conf = { exclude_rule_ids : Rule_ID.t list; severity : Rule.severity list }
+type products = SAST | SCA | Secrets | Interfile [@@deriving show]
```

Comment:
We have a product type already in semgrep_output_v1.atd. Might be better to reuse it
and maybe extend it with this new Interfile.
cc @IagoAbal